sdk/python/feast/feature_view.py

```diff
@@ -494,3 +494,22 @@ def most_recent_end_time(self) -> Optional[datetime]:
         if len(self.materialization_intervals) == 0:
             return None
         return max([interval[1] for interval in self.materialization_intervals])
+
+    @property
+    def online_store_ttl(self) -> Optional[int]:
+        """
+        Retrieves the online store TTL from the FeatureView's tags.
+
+        Returns:
+            An integer representing the TTL in seconds, or None if not set.
+        """
+        ttl_str = self.tags.get("online_store_ttl")
```

Comment:
Change the tag name to `online_store_key_ttl_seconds`. For easy to remember and align with feature_store.yaml configuration.